content/blog/2022/07/psp-migration-script.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Migrate your PSPs to Kubewarden policies!
+authors:
+- José Guilherme Vanz
+date: 2022-07-13
+---
+
+As announced in past [blog](https://www.kubewarden.io/blog/2022/01/mutating-policy-behave-as-validating/) [posts](https://www.kubewarden.io/blog/2022/05/psp-migration-docs/),
```

Comment:
We should consider putting this as a reference below instead of hyperlinking two words.